lib/events.js

```diff
@@ -180,10 +180,10 @@ api.getMergeEventPeers = async function({blockHeight, explain = false}) {
 // FIXME: remove this, no longer used
```

Comment:
Let's do this FIXME now and remove this whole function.

Reply:
We can also remove `aggregateHistory`. Removing these two methods also involves removing their mentions in tests, indexes, and `plugin.js`.

Reply:
Removed `events.getStartHash` here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity-storage/pull/15/commits/eaa5e45d7feb6298b3d8001eccbe5f665010792b

Removed `events.aggregateHistory` here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity-storage/pull/15/commits/ba42c0e8ed224e749a8fde6f78e2aeaf7da9bfdc